src/Microsoft.ML.Data/Transforms/Normalizer.cs

```diff
@@ -102,35 +102,35 @@ internal static ColumnOptionsBase Create(string outputColumnName, string inputCo
             }
         }
 
-        public abstract class FixZeroColumnOptionsBase : ColumnOptionsBase
+        public abstract class ControlZeroColumnOptionsBase : ColumnOptionsBase
         {
             public readonly bool FixZero;
 
-            private protected FixZeroColumnOptionsBase(string outputColumnName, string inputColumnName, long maxTrainingExamples, bool fixZero)
-                : base(outputColumnName, inputColumnName, maxTrainingExamples)
+            private protected ControlZeroColumnOptionsBase(string outputColumnName, string inputColumnName, long maximumExampleCount, bool fixZero)
```

Comment:
>ControlZeroColumnOptionsBase [](start = 30, length = 28)

Ditto above comment; I prefer FixZero.

Reply:
Zeros are not fixed. It floats around with different data sets.

---
In reply to: [264865051](https://github.com/dotnet/machinelearning/pull/2888#discussion_r264865051) [](ancestors = 264865051)